lib/alp_workloads/kvm_workload_utils.pm

```diff
@@ -0,0 +1,201 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This file provides the utility functions for kvm workload container.
+# Maintainer: alice <xlai@suse.com>, qe-virt@suse.de
+
+package alp_workloads::kvm_workload_utils;
+
+use base Exporter;
+use Exporter;
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures;
+use Utils::Backends qw(use_ssh_serial_console);
+use ipmi_backend_utils;
+use utils;
+use virt_autotest::virtual_network_utils qw(clean_all_virt_networks);
+
+our @EXPORT = qw(
+  set_ct_engine
+  set_kvm_image
+  pull_kvm_container
+  config_host_and_kvm_container
+  cleanup_host_and_kvm_container
+  start_kvm_container
+  enter_kvm_container_sh
+  exit_kvm_container
+  install_tools_within_kvm_container
+  setup_services_within_kvm_container
+  collect_kvm_container_setup_logs
+  setup_kvm_container_from_scratch
+  clean_and_resetup_kvm_container
+);
+
+# Container engine: can be podman(default), docker(not tested)
+our $ct_engine = "podman";
+
+my $DEFAULT_KVM_IMAGE = "registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/kvm:latest";
```

Comment:
I am not sure whether registry is going to stay the same. If not, I think you also need a variable like "$container_image_registry" ? Then $DEFAULT_KVM_IMAGE = $container_image_registry . "xxxx".

Reply:
I do not see the need now or near future. Besides, the registry info is already binded in kvm_image.